media/js/firefox/australis/growth-firstrun-test1.js

```diff
@@ -180,11 +175,9 @@
                 }
                 // oauth_complete sent after user verifies account
                 else if (event.data.indexOf('oauth_complete') > -1) {
-                    window.gaTrack(['_trackEvent', '/firstrun/ Optimization f35', 'Sign Up for Firefox Account', 'Verified Email Address']);
```

Comment:
How/where is this event being tracked now? Custom JS served by Google?

Anyway, if this `else if` condition is no longer needed, best to remove it.